### ISSUE TITLE
KeyError: 'ca-certificates-2019.11.28-hecc5488_0'

### ISSUE THREAD

**Issue by rsignell-usgs** [mariusvniekerk/conda-lock#12]
The following environment.yml fails to lock:
```yaml
name: pangeo
channels:
  - conda-forge
dependencies:
  - python
  - bokeh
  - cartopy
  - cython
  - dask
  - dask-cloudprovider
  - erddapy
  - numpy
  - python-blosc
  - hvplot
  - h5py
  - h5netcdf
  - ipykernel
  - ipywidgets
  - geoviews
  - holoviews
  - lz4
  - datashader
  - panel
  - metpy
  - netcdf4
  - xarray
  - s3fs
  - gcsfs
  - fsspec
  - utide
  - zarr
  - earthsim
```

```
sh-4.2$ conda-lock -f pangeo_env.yml -p linux-64
generating lockfile for linux-64
Traceback (most recent call last):
  File "/home/ec2-user/anaconda3/bin/conda-lock", line 8, in <module>
    sys.exit(main())
  File "/home/ec2-user/anaconda3/lib/python3.6/site-packages/conda_lock/conda_lock.py", line 124, in main
    platforms=args.platform or DEFAULT_PLATFORMS,
  File "/home/ec2-user/anaconda3/lib/python3.6/site-packages/conda_lock/conda_lock.py", line 84, in make_lock_files
    url = urls[pkg["dist_name"]]
KeyError: 'ca-certificates-2019.11.28-hecc5488_0'
sh-4.2$
```

**Comment by mariusvniekerk** [comment]
I managed to lock that environment just fine.

https://github.com/mariusvniekerk/conda-lock/blob/master/example/pangeo/conda-linux-64.lock

Can you share your `.condarc` and the conda version used to run the lock?

**Comment by rsignell-usgs** [comment]
my `.condarc`:
```
(base) sh-4.2$ more ~/.condarc
show_channel_urls: true
channel_priority: strict
channels:
  - conda-forge
envs_dirs:
  - /home/ec2-user/SageMaker/myenvs
pkgs_dirs:
  - ~/anaconda3/pkgs
auto_update_conda: false
safety_checks: disabled
```
conda version
```
(base) sh-4.2$ conda --version
conda 4.8.2
```

Today's error: 
```
(base) sh-4.2$ conda-lock -f pangeo_env.yml -p linux-64
generating lockfile for linux-64
Traceback (most recent call last):
  File "/home/ec2-user/anaconda3/bin/conda-lock", line 8, in <module>
    sys.exit(main())
  File "/home/ec2-user/anaconda3/lib/python3.7/site-packages/conda_lock/conda_lock.py", line 124, in main
    platforms=args.platform or DEFAULT_PLATFORMS,
  File "/home/ec2-user/anaconda3/lib/python3.7/site-packages/conda_lock/conda_lock.py", line 84, in make_lock_files
    url = urls[pkg["dist_name"]]
KeyError: '_libgcc_mutex-0.1-conda_forge'
```

**Comment by mariusvniekerk** [comment]
Can you post the output of 
```
conda config --show-sources
```

**Comment by rsignell-usgs** [comment]
```
(/home/ec2-user/SageMaker/myenvs/pangeo) sh-4.2$ conda config --show-sources
==> /home/ec2-user/.condarc <==
auto_update_conda: False
safety_checks: disabled
envs_dirs:
  - /home/ec2-user/SageMaker/myenvs
pkgs_dirs:
  - ~/anaconda3/pkgs
channel_priority: strict
channels:
  - conda-forge
show_channel_urls: True
```

**Comment by mariusvniekerk** [comment]
Okay this is caused by the pkgs_dirs portion of your config file.
I'll see if I can find a workaround, but this is a thorny conda config issur

**Comment by ocefpaf** [comment]
Fixed in #13.
